x-pack/test/security_solution_cypress/es_archives/alerts/mappings.json

```diff
@@ -9,31 +9,34 @@
     "index": ".siem-signals-default-000001",
     "mappings": {
       "dynamic": "false",
+      "_meta": {
+        "version": 3
+      },
       "properties": {
         "@timestamp": {
           "type": "date"
         },
         "agent": {
           "properties": {
             "ephemeral_id": {
-              "ignore_above": 1024,
```

Comment:
Do you know the cause behind all this churn? Our hope was that [consistent use](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/cypress/README.md#how-to-generate-a-new-archive) of es_archiver would minimize this sort of thing, but if its output is nondeterministic then that's worrisome.

Reply:
I copied and pasted this directly from the output of the mapping by doing a query rather than using es_archiver for this part. I don't think es_archiver sorts keys which is part of the problem.